tests/ts_cluster_create_table_via_sql.erl

```diff
@@ -26,30 +26,65 @@
 
 -export([confirm/0]).
 
+-define(TABLE, <<"t1">>).
+-define(CUSTOM_NVAL, 4).
+-define(CUSTOM_MVAL, <<"bo o''oo">>).
+
 confirm() ->
     [Node1, Node2 | _] = _Cluster = ts_util:build_cluster(multiple),
     C1 = rt:pbc(Node1),
     C2 = rt:pbc(Node2),
 
-    DDL = ts_util:get_ddl(),
-    Table = ts_util:get_default_bucket(),
+    CommonDDL =
```

Comment:
Instead of copying the DDL here to allow for table name customization, extend `ts_util`. This is a good feature for parallelization of tests. `get_ddl(Size, TableName)` e.g.